packages/integrations/cloudflare/README.md

```diff
@@ -107,6 +107,14 @@ export function get({ params }) {
 }
 ```
 
+## Headers, Redirects and function invocation routes
+
+Cloudflare has support for adding custom [headers](https://developers.cloudflare.com/pages/platform/headers/), configuring static [redirects](https://developers.cloudflare.com/pages/platform/redirects/) and defining which routes should [invoke functions](https://developers.cloudflare.com/pages/platform/functions/function-invocation-routes/). Cloudflare looks for `_headers`, `_redirects`, and `_routes.json` files in your build output directory to configure these features. This means they should be placed in your Astro project’s `public/` directory.
+
+### Custom `_routes.json`
+
+By default, `@astrojs/cloudflare` will generate a `_routes.json` file that lists all files from your `dist` folder in the respective exclude section. This will enable Cloudflare to serve files without a function invocation and therefore not count against the request limits of your Cloudflare plan. Creating a custom `_routes.json` will override this automatic optimization and, if not configured manually, cause function invocations and therefore count against the request limits of your Cloudflare plan.
```

Comment:
Couple of small language and style nits here, but looking good!

```suggestion
By default, `@astrojs/cloudflare` will generate a `_routes.json` file that lists all files from your `dist/` folder in the `exclude` array. This will enable Cloudflare to serve files without a function invocation. Creating a custom `_routes.json` will override this automatic optimization and, if not configured manually, cause function invocations that will count against the request limits of your Cloudflare plan.
```